js/src/components/modals/SemRushLoading.js

```diff
@@ -0,0 +1,31 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+import { Fragment } from "@wordpress/element";
+import { SvgIcon } from "@yoast/components";
+
+/**
+ * Creates the loading content for the SEMRush related keywords modal.
+ *
+ * @returns {React.Element} The SEMRush loading content.
+ */
+const SemRushLoading = () => {
+	return (
+		<Fragment>
+			<p>
+				{
+					sprintf(
+						/* translators: %1$s expands to "Yoast SEO", %2$s expands to "SEMrush". */
+						__( "Please wait while %1$s is connecting to %2$s to get related keyphrases...", "wordpress-seo" ),
+						"Yoast SEO",
+						"SEMRush"
+					)
+				}
+				&nbsp;
+				<SvgIcon icon="loading-spinner" />
```

Comment:
I asked architects whether we can use existing components that use `styled-components` internally. See conversation on Slack. In short: while there are no problems with this, the recommendation is to take a look at the component and see if it's easily doable to remove `styled-components` and turn it into a separate css file.

On trunk, there are already some icons in the `icons.css` file meant to be used as background. However, I'm not sure this would work for this specific icon because it has a couple keyframe animations.

Reply:
Wouldn't changing the component / moving the CSS be a job for the frontend team? This would also possibly impact more teams than just us, so it'd require a well-thought out solution. This also, in part, feels like scope creep to me.

Reply:
I'd agree and that was my first thought after architects' feedback. Anyways, we should carefully consider whether to use components that use `styled-components` and ping the frontend team.